### PR TITLE
Allow specifying a logo fallback string, or hiding it entirely

### PR DESCRIPTION
**Pull request by cmer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,6 +92,10 @@ You can pass the following options to generate a PDF:
     logo: "https://www.ruby-lang.org/images/header-ruby-logo@2x.png" # Downloaded with OpenURI
     ```
 
+  * `logo_fallback` - String to be displayed in place of a logo if no logo was specified. - _Optional, defaults to company name_
```
Comment (excid3) [inline README.md:95]: What if we allow the `logo` option to just be a string? 

We can check if the `logo` includes `://` which would hint that it's a URL of some sort. If it doesn't include that, we can just render out the string. That way we don't have to introduce a new option?

**Comment by cmer** [comment]
Great idea! I made the changes as proposed. I also added tests for it because it is getting pretty complex!

File: lib/receipts/base.rb [hunk 2]
```diff
@@ -36,23 +36,21 @@ def setup_fonts(custom_font = nil)
...
     def header(company: {}, height: 16)
       logo = company[:logo]
 
-      if logo.nil?
-        text company.fetch(:name), align: :right, style: :bold, size: 16, color: "4b5563"
-      else
+      if logo_is_image?(logo)
         image load_image(logo), height: height, position: :right
+        move_up height
+      elsif logo.nil? || logo != false
```
Comment (DaniG2k) [inline lib/receipts/base.rb:48]: Could probably even just do

```suggestion
      elsif logo.nil? || logo
```

File: lib/receipts/base.rb [hunk 3]
```diff
@@ -36,23 +36,21 @@ def setup_fonts(custom_font = nil)
...
+        str = logo || company.fetch(:name)
+        text str, align: :right, style: :bold, size: 16, color: "4b5563"
```
Comment (DaniG2k) [inline lib/receipts/base.rb:50]: Would be sweet if we could pass in a logo size. Any reason why we can't do this? 🤔

File: lib/receipts/base.rb [hunk 4]
```diff
@@ -36,23 +36,21 @@ def setup_fonts(custom_font = nil)
...
     end
 
     def header(company: {}, height: 16)
       logo = company[:logo]
 
-      if logo.nil?
-        text company.fetch(:name), align: :right, style: :bold, size: 16, color: "4b5563"
-      else
+      if logo_is_image?(logo)
         image load_image(logo), height: height, position: :right
```
Comment (DaniG2k) [inline lib/receipts/base.rb:46]: Would be sweet if we could pass in a logo height. Any reason why we can't do this? 🤔

**Pull request closed without merging**